### PR TITLE
BETA: Register dave9123.is-a.dev

### PR DESCRIPTION
**Pull request by dave9123**
Added `dave9123.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/dave9123.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "dave9123",
+        "email": "dave.turmawan@outlook.com"
+    },
+    "record": {
+        "CNAME": "dave9123-isadev.netlify.app"
```
Comment (DEV-DIBSTER) [inline domains/dave9123.json:7]: We do not support netlify.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Comment by dave9123** [comment]
What should I use then?

**Comment by dave9123** [comment]
I think I messed up

**Comment by dave9123** [comment]
Can I use this?

File: domains/dave9123.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "dave9123",
+        "email": "dave.turmawan@outlook.com"
+    },
+    "record": {
+        "CNAME": "dave9123.github.io/dave9123_page"
```
Comment (DEV-DIBSTER) [inline domains/dave9123.json:7]: Invalid CNAME, cannot store / within CNAME.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/dave9123.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "dave9123",
+        "email": "dave.turmawan@outlook.com"
+    },
+    "record": {
+        "CNAME": "dave9123.github.io"
```
Comment (DEV-DIBSTER) [inline domains/dave9123.json:7]: CNAME still redirects to this domain, which would redirect back to your GitHub page, creating a loop.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Comment by DEV-DIBSTER** [comment]
Yes, but your site content doesn't exist. All it's doing is redirecting.

**Comment by DEV-DIBSTER** [comment]
Infact I don't see your site repository, are you sure you made that public?

**Comment by dave9123** [comment]
Because it's waiting for CNAME record
________________________________

**Comment by dave9123** [comment]
I have GitHub Pro, it's made private
________________________________

**Comment by dave9123** [comment]
:)

**Review by DEV-DIBSTER — APPROVED** [review]

**Comment by dave9123** [comment]
Thanks so much

**Pull request merged**